### DISCUSSION TITLE
O Atom  não permanece instalado na minha maquina

### DISCUSSION THREAD

**Discussion by nic690** [frontendbr/forum#2201] [Perguntas e Respostas]
Toda vez que tenho que usar o Atom eu tenho que instalar ele de novo no arquivo executável. Alguém sabe como resolver ? 

lembrando que estou usando Windows 10

**Comment by expalmer** [comment]
Sim, usa o VS Code

**Reply by nic690** [reply]
Eu usava mas eu prefiro o Atom
